.travis.yml

```diff
@@ -8,11 +8,10 @@ php:
   - nightly
 
 install:
-  - wget https://phar.phpunit.de/phpunit-old.phar
   - composer install
 
 script:
-  - php phpunit-old.phar
```

Comment:
I don't want PHPUnit in muh dependencies. Just replace `phpunit-old.phar` for `phpunit-5.7.phar` and drop support for PHP 5.5 in the .travis.yml file.

It's either that or remove support for the nightly builds, since PHP 7.2 cannot run PHPUnit 4.8. And I think it makes more sense to drop support for the dead version.

Reply:
@1ma